lib/fathead/node_js/parse.js

```diff
@@ -6,13 +6,27 @@ var fs = require('fs')
     items = [];
 
 function createItem(ops) {
-    items.push([
-        ops.name,
-        'A',
-        '','','','','','','','','',
-        ops.abstract,
-        URL_BASE + ops.URL
-    ].join('\t'));
+    //If item contains any spaces, make it a R item 
+    //for the corresponding A, otherwise, add it as a A item 
+    if (ops.name.indexOf(' ') > -1) {
+        var redirect = ops.name.replace(' ', '.');
+
+        items.push([
+            ops.name,
+            'R',
+            redirect,
+            '',
```

Comment:
@jarrodthibodeau yes, it's missing the ''s,  so that you have 13 tabs [(13 fields)](https://docs.duckduckhack.com/resources/fathead-overview.html#output-fields) in the end. Thanks